packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/app/config/app_constants.dart

```diff
@@ -4,8 +4,41 @@
 
 // Here lie all your app constants
 
-/// Vapid Key is required in order to for FCM to run on web properly
-/// https://github.com/FirebaseExtended/flutterfire/blob/4c9b5d28de9eeb5ce76c856fbd0c7b3ec8615e45/docs/messaging/usage.mdx#web-tokens
+/// The [isInTestMode] in these files is used to exclude the dependencies
+/// from smart widgets for golden tests.
+///
+/// Since we don't want to add dependency on [universal_io/io.dart],
```

Comment:
I believe this documentation deserves its own place in docs folder  (docs/golden_tests.md) with a reference in the main README.md file (Golden tests section) here we can leave only a reference to the MD file

Reply:
Done